### PR TITLE
Remove outdated bug bounty program docs

### PR DESCRIPTION
**Pull request by matkam**
Updated info can be found here: https://help.medium.com/hc/en-us/articles/213481308-Bug-Bounty-Disclosure-Program

<details>
<summary>REVIEWERS (4)</summary>

R=@valueof
R=@KCarpen
R=@lseljak92
R=@nichellebot
</details>

### CODE REVIEW

**Review by nichelle-hall — APPROVED** [review]

File: bounty-program-exclusions.md [hunk 1]
```diff
@@ -1,14 +0,0 @@
-# Exclusions from bug bounty eligibility :/
```
Comment (Baro1905) [inline bounty-program-exclusions.md:1]: bounty-program-exclusions.md

**Comment by alwerr** [comment]
Its look like the terms in here older than the one in the medium website. Can one use the terms in the website? Are they  Creative Commons Sharealike license too?

**Comment by gueldner** [comment]
> Its look like the terms in here older than the one in the medium website. Can one use the terms in the website? Are they Creative Commons Sharealike license too?

@alwerr can you send us an email to trust@medium.com? We can discuss this there. Thanks!

**Comment by alwerr** [comment]
Sure, Thanks!

File: bounty-program-exclusions.md (hunk 1 above)
Comment (abene130) [reply bounty-program-exclusions.md:1]: 05c5eb1ded0e76ea2efd2ce1b3bd8e61b7539398

**Pull request merged**